### ISSUE TITLE
Error reporting not working for single invalid property

### ISSUE THREAD

**Issue by mjbvz** [Microsoft/typescript-styled-plugin#20]
```js
import styled from 'styled-components'

styled.button`
    boarder: 1px solid hotpink; 
`
```

Should show an error since `boarder` is not a known property name

**Comment by utkuturunc** [comment]
```
const MainLayout = styled(Layout)`
  min-height: 100vh;
  boarder: 100vh;
`
```

this also will not show an error. if i change any letter in min-height, the plugin is able to detect it.

**Comment by mjbvz** [comment]
Should be fixed in 0.1.2:

![screen shot 2017-10-24 at 4 23 52 pm](https://user-images.githubusercontent.com/12821956/31972867-c14b1bcc-b8d7-11e7-832d-ce49a3868c34.png)
